simplecpp.cpp

```diff
@@ -1737,16 +1737,10 @@ namespace {
         }
     }
 
+    const char * const altopData[] = {"and","or","bitand","bitor","not","not_eq","xor"};
+    const std::set<std::string> altop(&altopData[0], &altopData[7]);
```

Comment:
Limitation is conservative. It shouldn't have higher requirements than cppcheck (https://github.com/danmar/cppcheck).